driver-core/src/main/java/com/datastax/driver/core/CBUtil.java

```diff
@@ -113,22 +113,17 @@ public static int sizeOfLongString(String str) {
         }
     }
 
-    public static void writeBytes(byte[] bytes, ByteBuf cb) {
+    public static void writeShortBytes(byte[] bytes, ByteBuf cb) {
         cb.writeShort(bytes.length);
         cb.writeBytes(bytes);
     }
 
-    public static void writeBytes(ByteBuffer bytes, ByteBuf cb) {
```

Comment:
Removed unused methods, while they might be useful in future, would be easy to reimplement them.